Binary files wxWidgets.orig/buildMac/.pch/wxprec_advdll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_advdll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_auidll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_auidll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_coredll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_coredll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_htmldll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_htmldll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_mediadll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_mediadll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_propgriddll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_propgriddll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_qadll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_qadll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_ribbondll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_ribbondll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_richtextdll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_richtextdll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_stcdll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_stcdll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_webviewdll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_webviewdll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/.pch/wxprec_xrcdll/wx/wxprec.h.gch and wxWidgets/buildMac/.pch/wxprec_xrcdll/wx/wxprec.h.gch differ
Binary files wxWidgets.orig/buildMac/coredll_colrdlgg.o and wxWidgets/buildMac/coredll_colrdlgg.o differ
Binary files wxWidgets.orig/buildMac/coredll_slider_osx.o and wxWidgets/buildMac/coredll_slider_osx.o differ
Binary files wxWidgets.orig/buildMac/coredll_slidercmn.o and wxWidgets/buildMac/coredll_slidercmn.o differ
Binary files wxWidgets.orig/buildMac/coredll_utilscmn.o and wxWidgets/buildMac/coredll_utilscmn.o differ
Binary files wxWidgets.orig/buildMac/coredll_valgen.o and wxWidgets/buildMac/coredll_valgen.o differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_adv-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_adv-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_adv-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_adv-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_adv-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_adv-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_aui-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_aui-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_aui-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_aui-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_aui-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_aui-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_core-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_core-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_core-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_core-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_core-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_core-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_html-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_html-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_html-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_html-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_html-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_html-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_media-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_media-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_media-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_media-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_media-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_media-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_propgrid-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_propgrid-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_propgrid-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_propgrid-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_propgrid-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_propgrid-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_qa-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_qa-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_qa-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_qa-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_qa-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_qa-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_ribbon-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_ribbon-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_ribbon-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_ribbon-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_ribbon-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_ribbon-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_richtext-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_richtext-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_richtext-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_richtext-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_richtext-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_richtext-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_stc-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_stc-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_stc-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_stc-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_stc-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_stc-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_webview-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_webview-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_webview-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_webview-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_webview-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_webview-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_xrc-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_xrc-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_xrc-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_xrc-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_xrc-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_xrc-3.1.dylib differ
Binary files wxWidgets.orig/buildMac/samples/widgets/widgets and wxWidgets/buildMac/samples/widgets/widgets differ
Binary files wxWidgets.orig/buildMac/samples/widgets/widgets.app/Contents/MacOS/widgets and wxWidgets/buildMac/samples/widgets/widgets.app/Contents/MacOS/widgets differ
Binary files wxWidgets.orig/buildMac/samples/widgets/widgets_slider.o and wxWidgets/buildMac/samples/widgets/widgets_slider.o differ
Binary files wxWidgets.orig/buildMac/xrcdll_xh_sizer.o and wxWidgets/buildMac/xrcdll_xh_sizer.o differ
Binary files wxWidgets.orig/buildMac/xrcdll_xh_slidr.o and wxWidgets/buildMac/xrcdll_xh_slidr.o differ
diff -bru wxWidgets.orig/include/wx/osx/slider.h wxWidgets/include/wx/osx/slider.h
--- wxWidgets.orig/include/wx/osx/slider.h	2015-07-04 18:31:06.000000000 -0400
+++ wxWidgets/include/wx/osx/slider.h	2015-07-06 00:15:49.000000000 -0400
@@ -86,6 +86,7 @@
         list.push_back( m_macMaximumStatic );
         return list;
     }
+    void OnMouseDown(wxMouseEvent &event);
     
     // Platform-specific implementation of SetTickFreq
     virtual void DoSetTickFreq(int freq);
diff -bru wxWidgets.orig/src/osx/slider_osx.cpp wxWidgets/src/osx/slider_osx.cpp
--- wxWidgets.orig/src/osx/slider_osx.cpp	2015-07-04 14:09:01.000000000 -0400
+++ wxWidgets/src/osx/slider_osx.cpp	2015-07-06 00:23:47.000000000 -0400
@@ -120,15 +120,18 @@
     if (style & wxSL_LABELS)
     {
         m_macMinimumStatic = new wxStaticText( parent, wxID_ANY, wxEmptyString );
+        m_macMinimumStatic->Bind( wxEVT_LEFT_DOWN, &wxSlider::OnMouseDown, this );
         m_macMaximumStatic = new wxStaticText( parent, wxID_ANY, wxEmptyString );
+        m_macMaximumStatic->Bind( wxEVT_LEFT_DOWN, &wxSlider::OnMouseDown, this );
         m_macValueStatic = new wxStaticText( parent, wxID_ANY, wxEmptyString );
+        m_macValueStatic->Bind( wxEVT_LEFT_DOWN, &wxSlider::OnMouseDown, this );
     }
 
     SetRange(minValue, maxValue);
     SetValue(value);
 
     MacPostControlCreate(pos, size);
-
+    Bind( wxEVT_LEFT_DOWN, &wxSlider::OnMouseDown, this );
     return true;
 }
 
@@ -304,6 +307,11 @@
     HandleWindowEvent( cevent );
 }
 
+void wxSlider::OnMouseDown(wxMouseEvent &event)
+{
+    SetFocus();
+}
+
 bool wxSlider::OSXHandleClicked( double WXUNUSED(timestampsec) )
 {
     TriggerScrollEvent(wxEVT_SCROLL_THUMBRELEASE);
